src/account/models/updated-project.model.ts

```diff
@@ -30,6 +31,9 @@ export class UpdatedProject {
     })
     @IsOptional()
     @IsString()
+    @MaxLength(80, {
```

Comment:
thought(blocking): We need to talk about the max length. Inside the database we allow a much higher value, which is probably wrong. How much chars can we display on the cards at the frontend?

Reply:
As we discussed internally we will stick on 80 characters for now. Import script and database will continue to support larger values.